.github/workflows/windows.yml

```diff
@@ -0,0 +1,54 @@
+name: windows
+
+on: [push, pull_request]
+
+jobs:
+  build_win_msvc:
+    name: MSVC w/o MPI
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      name: Install Python
+      with:
+        python-version: '3.8'
+    - name: Build & Test
+      run: |
+        md build
+        cd build
+        powershell.exe ..\share\openPMD\download_samples.ps1
+        cmake .. `
+              -DCMAKE_BUILD_TYPE=Debug    `
+              -DCMAKE_VERBOSE_MAKEFILE=ON
```

Comment:
Does this miss a `cmake --install .` step? The Python bindings are not found by the CI. Same below.
Also, if the test's name is "w/o MPI", maybe explicitly set `-DopenPMD_USE_MPI=OFF`?

Reply:
Usually, I did set up the `ctest` environment to find the build export module in the build directory. But there seems to be something off with that hint indeed.